pkg/registry/authentication/tokenreview/storage.go

```diff
@@ -44,6 +45,7 @@ func (r *REST) Create(ctx genericapirequest.Context, obj runtime.Object, include
 	if !ok {
 		return nil, apierrors.NewBadRequest(fmt.Sprintf("not a TokenReview: %#v", obj))
 	}
+	validation.ValidateTokenReview(tokenReview)
```

Comment:
I'd probably inline the length check on the spec.token rather than adding a whole package just to check that one field.